packages/symbols/rollup.config.js

```diff
@@ -0,0 +1,30 @@
+import babel from 'rollup-plugin-babel'
+import pkg from './package.json'
+
+const makeExternalPredicate = externalArr => {
```

Comment:
should we extract a function into another build-helper file?

Reply:
ideally I'd like to just drop low-level rollup for microbundle (its a rollup wrapper) in the future, so wouldnt bother with that, just need to find some time to contribute to microbundle more to cover more advanced use cases 😅

Reply:
should we create appropriate issue?

Reply:
its not rly actionable right now, i would only like to move to microbundle if it will be able to tackle all the required tasks with simpler config (so config doesnt have to be updated in multiple places, which I tend to do a lot :/)

Reply:
ok. let's then move the helper in one place. later we can migrate to other builder

Reply:
not sure how to share such a helper appropriately between packages without making it a package (and i dont think its worth to make it a package), and imho we shouldnt reach out to anything outside a single package directory - that would break package encapsulation